src/vstart.sh

```diff
@@ -455,8 +451,22 @@ if [ $kill_all -eq 1 ]; then
     $SUDO $INIT_CEPH stop
 fi
 
-if [ "$overwrite_conf" -eq 0 ]; then
-    CEPH_ASOK_DIR=`dirname $($CEPH_BIN/ceph-conf  -c $conf_fn --show-config-value admin_socket)`
+if [ "$new" -ne 0 ]; then
```

Comment:
why are you transposing these two branches?

Reply:
I just thought of retaining the earlier check ["$new" -ne 0' ]. Fixed it.